index.html

```diff
@@ -82,7 +82,38 @@ <h1 class="worktext">My Recent Works</h1>
                <li> <a href="#">Get My Resume</a></li>
             </ul>
         </div>
```

Comment:
- [ ] I noticed that your project only has one section, and since we are imitating the figma design and your design and the figma one are very dissimilar, kindly add additional sections to match the figma design, I encourage you to follow the figma design [have a look on how your project should look like](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) :wink: 

Figma Design :point_down: 
![top3](https://user-images.githubusercontent.com/88894525/199611323-dc4a87b3-4279-4a68-a518-25bbdf26501b.png)
![top2](https://user-images.githubusercontent.com/88894525/199611329-e4ab3eac-885b-4b54-8acd-dbedf3d10e70.png)


Your Design :point_down: 
![top1](https://user-images.githubusercontent.com/88894525/199611409-83168f1c-8314-47fa-a394-74008dd20531.png)

Reply:
@Alfredbis29 please note that we are reviewing the contact form that we did with my partner.